mongodbatlas/provider.go

```diff
@@ -60,6 +62,11 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("MONGODB_REALM_BASE_URL", ""),
 				Description: "MongoDB Realm Base URL",
 			},
+			"is_mongodbgov_cloud": {
```

Comment:
Now that we have this we need to add a section to this page: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/index.html.markdown

## Configure MongoDB Atlas for Government 
In order to enable the Terraform MongoDB Atlas Provider for use with [MongoDB Atlas for Government](https://www.mongodb.com/docs/atlas/government/api/#creating-a-project) add `is_mongodbgov_cloud = true` to your provider configuration: 

```terraform
# Configure the MongoDB Atlas Provider for MongoDB Atlas for Government
provider "mongodbatlas" {
  public_key = var.mongodbatlas_public_key
  private_key  = var.mongodbatlas_private_key
  is_mongodbgov_cloud = true
}
# Create the resources
```

Reply:
Added gov configuration section